src/V3Ast.cpp

```diff
@@ -1141,6 +1141,47 @@ void AstNode::dumpTreeFile(const string& filename, bool append, bool doDump, boo
     }
 }
 
+static void drawEdge(std::ostream& os, const AstNode* n1,  const AstNode* n2, const std::string& name) {
```

Comment:
I will use `thisp` to replace `n1`, due to there is another local var "nodep" in the for loop. For this function, it is to draw a connection/edge only, instead of the children node. I will use drawEdge() & edgeName here.

`static void drawEdge(std::ostream& os, const AstNode* thisp, const AstNode* childp, const std::string& edgeName) {`